listenbrainz/config.py.sample

```diff
@@ -22,10 +22,8 @@ SQLALCHEMY_TIMESCALE_URI = "postgresql://listenbrainz_ts:listenbrainz_ts@lb_db/l
 TIMESCALE_ADMIN_URI = "postgresql://postgres:postgres@lb_db/postgres"
 TIMESCALE_ADMIN_LB_URI = "postgresql://postgres:postgres@lb_db/listenbrainz_ts"
 
-# MessyBrainz
-MESSYBRAINZ_SQLALCHEMY_DATABASE_URI = "postgresql://messybrainz:messybrainz@lb_db:5432/messybrainz"
-MESSYBRAINZ_ADMIN_URI = "postgresql://postgres:postgres@lb_db/postgres"
-MESSYBRAINZ_ADMIN_MSB_URI = "postgresql://postgres:postgres@lb_db/messybrainz"
+# MetaBrainz
+SQLALCHEMY_METABRAINZ_URI = ""
```

Comment:
We should probably add a comment as to why this is where and when someone might need to fill it out.

Reply:
done